query-node/mappings/src/content/nft.ts

```diff
@@ -349,12 +351,25 @@ export async function createNft(
     : undefined
   const decodedMetadata = nftIssuanceParameters.nft_metadata.toString()
 
+  // Is NFT owned by channel or some member
+  const isOwnedByChannel = !ownerMember
+
+  // load channel
+  const channel = (await getById(store, Video, video.getId(), ['channel'])).channel
+
+  // load channel ownerCuratorGroup (if any)
+  const ownerCuratorGroup = isOwnedByChannel
+    ? (await getById(store, Channel, channel.getId(), ['ownerCuratorGroup'])).ownerCuratorGroup
+    : undefined
+
```

Comment:
Let's get rid of both `getById`s here. We can rely on having these values loaded in `video` (`video.channel`, `video.channel.ownerCuratorGroup`) - see https://github.com/Joystream/joystream/pull/3405/files#r827498072

Since `ownerCuratorGroup` load is not ensured yet, ensure it is loaded before `createNft` is called. There are only 2 places right now - in `contentNft_NftIssued` and `content_VideoCreated`.

Also, don't forget that channel can also be owned by a member. Only the curator group owner is handled at the moment.